src/exm-upgrade-assistant.c

```diff
@@ -595,7 +595,7 @@ populate_drop_down (ExmUpgradeAssistant *self)
         gchar *key;
 
         key = g_strdup_printf ("GNOME %d", index);
-        g_hash_table_insert (hash_table, key, index);
+        g_hash_table_insert (hash_table, key, &index);
```

Comment:
This should work with the new version and older ones:

```suggestion
        g_hash_table_insert (hash_table, key, GINT_TO_POINTER (index));
```

It requires changing

https://github.com/mjakeman/extension-manager/blob/d146b8929de7a0fbde4300aef8bc3c9831c4044f/src/exm-upgrade-assistant.c#L409

to `target_version = GPOINTER_TO_INT (g_hash_table_lookup (self->version_map, key));`

Reply:
Did so, seems to work fine.
![image](https://github.com/mjakeman/extension-manager/assets/21365918/e05f02a5-4eab-4ea2-b14a-a751e37db4f7)